### DISCUSSION TITLE
Q&A in next Quarkus Insights

### DISCUSSION THREAD

**Discussion by maxandersen** [quarkusio/quarkus#26740] [Events]
We're having a Q&A session on Monday's Quarkus Insights (July 18th 2022). 

Post your question in this thread or use the #quarkusinsights tag on twitter and linkedin.

We will then do our best to answer them on Monday's Quarkus Insights episode!

**Comment by markuseckstein** [comment]
Hi Quarkus Team,

I wonder if OpenTelemetry instrumentation attribute `@WithSpan` as described in https://quarkus.io/guides/opentelemetry#cdi also works for methods returning an `Uni`?

My impression is that methods returning an `Uni` get traced, but are somewhat not connnected with the previous trace context.

Am I doing anything wrong or are methods returning Unis are simply not supported?

best regards,
Markus

**Comment by stefanwendelmann** [comment]
Thanks for having the Q&A Session.

Is HTTP/3 considered / in dev / in release ?

best regards
Stefan

**Comment by himanshumps** [comment]
We all have seen good benchmarks of TTFB after server startup. Also some good benchmarks on techempower. All these tests are on a very small response (around 1024 bytes) and on non-secure endpoints. Can we see how efficient quarkus is when running the benchmarks for real world scenarios  like 5Kb, 10Kb, 20KB and 1MB response sizes for an extended period of time over SSL(TLS v1.2) over a period of 15-20 minutes?

**Comment by sekaijin** [comment]
Hello

In many cases it is necessary to execute code before the execution of the application. This is the case for example of Liquibase which will initialize or update the database before running the application.
But this is also found in many other cases and for various technologies.
Today every extension that needs to set up such a process does it with ad-hoc code.
I wonder if it would not be interesting to add to Quarkus a framework (api, annotation...) allowing all the creator of extension but also the developer of application to simply add such a process.
